readme.md

```diff
@@ -42,7 +42,18 @@ That way you are in control of the order in which things are executed.
 
 
 ## Testing
+There should be unit tests for your code. A unit test should be independent and do an assert for each unit. The tests for business logic should be independent so it has no dependencies on databases or external services. An exception is a repository test -- there you want to test a database transaction.
```

Comment:
Do we have a set structure that we follow for our unit tests?

Reply:
Arrange, Act, Assert